README.md

```diff
@@ -14,17 +14,91 @@ The `ballerinax/spotify` package provides APIs to connect and interact with Spot
 
 ## Setup guide
 
-[//]: # (TODO: Add detailed steps to obtain credentials and configure the module.)
+To use the Spotify connector, you must have access to the Spotify API through a [Spotify developer account](https://developer.spotify.com) and a project under it. If you do not have a Spotify Developer account, you can sign up for one [here](https://developer.spotify.com/documentation/web-api/concepts/access-token).
+
+### Step 1: Create a Spotify Developer Project
+
+1. Open the [Spotify Developer Portal](https://developer.spotify.com/dashboard).
+
+<img src=https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-spotify/main/docs/setup/resources/app_overview.png alt="Spoify Developer Portal" style="width: 70%;">
+
+2. Click on the "Create an app" tab and select an existing project or create a new one for which you want API Keys and Authentication Tokens.
+
+<img src=https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-spotify/main/docs/setup/resources/createappdialog.png alt="Spoify Create App" style="width: 70%;">
+
+### Step 2: Set up user authentication settings
+
+<img src=https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-spotify/main/docs/setup/resources/dashboardeditsettings.png alt="Spoify Dashboard edit settings" style="width: 70%;">
+
+Enter an App Name and App Description of your choice (they   will be displayed to the user on the grant screen), put a tick in the Developer Terms of Service checkbox and finally click on CREATE. Your application is now registered, and you'll be redirected to the app overview page.
+
+### Step 3. Obtain Client Id and Client Secret.
+
+With our credentials in hand, we are ready to request an access token.
+
+1. Send a POST request to the token endpoint URI.
+2. Add the Content-Type header set to the application/x-www-form-urlencoded value.
+3. Add a HTTP body containing the Client ID and Client Secret, along with the grant_type parameter set to client_credentials.
+
+```bash
+    curl -X POST "https://accounts.spotify.com/api/token" \
+     -H "Content-Type: application/x-www-form-urlencoded" \
+     -d "grant_type=client_credentials&client_id=your-client-id&client_secret=your-client-secret"
+```
+The response will return an access token valid for 1 hour
+```
+{
+  "access_token": "BQDBKJ5eo5jxbtpWjVOj7ryS84khybFpP_lTqzV7uV-T_m0cTfwvdn5BnBSKPxKgEb11",
```

Comment:
Just add 
```suggestion
  "access_token": "<Access Token>",
```
And this should be enough. I saw a screenshot also with some client details, lets blur/hide them.